### ISSUE TITLE
Attempts to use any address while loading example.py fail

### ISSUE THREAD

**Issue by unromen** [huacnlee/PokemonGoMove#39]
Attempts to use a street address (any that I have tried) return the following error:

[-] retrying_set_location: geocoder exception (The given key has gone over the requests limit in the 24 hour period or has submitted too many requests in too short a period of time.), retrying

I have attempted using the default API key in credentials.json, as well as several newly generated keys from personal accounts. All Google API keys return the same error. Please advise

**Comment by RyanBeckett** [comment]
You need to use your own Google Maps API - do you know how todo this?

**Comment by unromen** [comment]
Yes, the key has been changed to several of my own API keys - in fact, using no key at all returns the exact same error as using any API whatsoever

**Comment by we-do** [comment]
I am also having no luck using my own credentials (tried 4 different keys, new ones and old ones)... are the keys being set anywhere other than the credentials.json file?

**Comment by unromen** [comment]
The only other place I could find it was the last edit made to the project - putting my own key here helped by fixing the top map, but seems to have done nothing in the long term for this issue specifically.

**Comment by fengyuhan** [comment]
try using 50,-104

**Comment by unromen** [comment]
Using coordinates works, but using an address does not - not the biggest deal in the world because everything still functions fairly normally

**Comment by jk2K** [comment]
You need to enable the following api
![image](https://cloud.githubusercontent.com/assets/4025839/17165869/082e7012-5406-11e6-91ac-2cd82026bcb9.png)

and change line 171 in `example.py`
```python
geolocator = GoogleV3(GOOGLEMAPS_KEY)
```

I have create a PR, https://github.com/huacnlee/PokemonGoMove/pull/67
